README.md

```diff
@@ -1 +1,360 @@
+
+### API protocols
+
+In order to be authorized to use any of the payments API endpoints, you will need to send your api_key in the header of your request.
+
+
+### API hosts
+
+service.centrapay.com/payments/api
+
+### Endpoints
+
+|             |                                                                                                   |
+|-------------|---------------------------------------------------------------------------------------------------|
+| Requests    | /requests.cancel <br> /requests.create <br> /requests.info <br> /requests.pay <br> /requests.void |
+| Service     | /service.info                                                                                     |
+| Transaction | /transactions.processSettlements <br> /transactions.refund                                        |
+
+### QR code payments
+
+#### Merchant Push
+
+At a high level, a merchant makes a **POST** request to `/requests.create` and optionally sets notifyURL to the URL that will receive webhook notifications when the request status is updated. 
+
+A response will be sent to the merchant with a payment request and accompanying QR code (as a base64 encoded string) that encodes the requestId, which can then be presented to consumers as the merchant sees fit.
+
+ A consumer then is then presented with the ID or QR code, which they then scan to initiate payment. If the payment is successful, a notification is sent to the webhook (if provided), confirming with the merchant that the consumer has completed the payment request.
+
+#### Merchant Pull
+
+Where the merchant does not have a screen or payment terminal to display the QR code, this method works well by allowing merchants to simply scan a 2D barcode which is presented on the consumers device to accept payment.
+
+Steps: 
+1. Generate barcode in Sylo 
+2. Create a request in requests.create, using fields: 
+3. Amount 
+4. Asset 
+5. MerchantID 
+6. PatronCode (Enter barcode) 
+
+>Once you have created the request, the apk should display payment option/ confirmation. 
+
+Example:
+
+An example URL that a **POST** request could be sent to would be the following:
+
+``` 
+https://service.centrapay.com/payments/api/requests.create?amount=100&asset=NZD&merchantId=1399b053-b3dd-4c5b-9859-b5bf5c2ac477&patronCode=2368720114399939″-H“accept:application/json”-H“x-api-key:1399b053-b3dd-4c5b-9859-b5bf5c2ac477
+```
+
+#### Non integrated
+
+Where the merchant is unable to integrate electronically, Centrapay can issue the merchant with a unique QR code which can be used at the point of sale to engage consumers and receive payment.
+
+
+## Im not sure about this section
+
+### [Creating a payment request](https://service.centrapay.com/payments/api/documentation#/Requests/postRequestscreate)
+
+
+A payment request can be created by sending a **POST** request to the following endpoint:<br>
+https://service.centrapay.com/payments/api/requests.create
+
+The following parameters **must** be provided with the request:
```

Comment:
Maybe just have this as Required Parameters and Optional parameters instead of `The following parameters **must** be provided with the request:` and `The following parameters can **optionally** be provided with the request:`